content/recipes/nestia.md

```diff
@@ -0,0 +1,698 @@
+
+Looking at the another file [generic/swagger.json](https://editor.swagger.io/?url=https%3A%2F%2Fraw.githubusercontent.com%2Fsamchon%2Fnestia%2Fmaster%2Fdemo%2Fgeneric%2Fswagger.json), you can find that there isn't any problem even when the generic typed DTO interfaces or controller classes come. Furthermore, traveling the `description` properties of the [generic/swagger.json](https://editor.swagger.io/?url=https%3A%2F%2Fraw.githubusercontent.com%2Fsamchon%2Fnestia%2Fmaster%2Fdemo%2Funion%2Fswagger.json), comments written on the generic interfaces and controllers are exactly revived in each route functions and schema definitions.
+
+  - View in the `Swagger Editor`
+    - [simple/swagger.json](https://editor.swagger.io/?url=https%3A%2F%2Fraw.githubusercontent.com%2Fsamchon%2Fnestia%2Fmaster%2Fdemo%2Fsimple%2Fswagger.json)
+    - [generic/swagger.json](https://editor.swagger.io/?url=https%3A%2F%2Fraw.githubusercontent.com%2Fsamchon%2Fnestia%2Fmaster%2Fdemo%2Fgeneric%2Fswagger.json)
+    - [union/swagger.json](https://editor.swagger.io/?url=https%3A%2F%2Fraw.githubusercontent.com%2Fsamchon%2Fnestia%2Fmaster%2Fdemo%2Funion%2Fswagger.json)
+
+<img src="https://github.com/samchon/nestia/wiki/images/swagger-editor-comment.png" style="max-width: 100%" />
+
+
+
+
+#### Configuration
+<br/>
+<img src="https://github.com/samchon/nestia/wiki/images/configuration-comparison-table.png" style="max-width: 100%" />
+
+You can configure the nestia builder options by two ways: `CLI` or `configuration file`.
+
+The CLI (Command Line Interface) is easy to use, because it does not require any type of configuration file wrinting. However, it's not safe thing that repeating the same command whenever generating the SDK or `swagger.json` through the CLI, because command miswriting can be happened.
+
+```bash
+# BASIC COMMAND
+npx nestia <nestia|swagger> <source_directories_or_patterns> \
+    --exclude <exclude_directory_or_pattern> \
+    --out <output_directory_or_file>
+
+# EXAMPLES
+npx nestia nestia "src/controllers" --out "src/api"
+npx nestia swagger "src/**/*.controller.ts" -- out "swagger.json"
+npx nestia swagger "src/main/controllers" "src/sub/controllers" \
+    --exclude "src/main/test" \
+    --out "composite.swagger.json"
+
+# ONLY WHEN NESTIA.CONFIG.TS EXISTS
+npx nestia nestia
+npx nestia swagger
+```
+
+Otherwise, configuration file `nestia.config.ts` may hard to construct and the construction would be annoying and inconvenient, such configuration would be useful within framework of the reusability. Also, the configuration file `nestia.config.ts` use much detailed options than the `CLI` and its content can be ensured its safety by TypeScript compiler through the `IConfigruation` interface type.
```

Comment:
The sentence structure here doesn't quite make sense. I can't get a clear idea of what I'm reading to make a suggestion either.